packages/dd-trace/src/appsec/iast/analyzers/intermendiate-cookies-analyzer.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+const Analyzer = require('./vulnerability-analyzer')
+const dc = require('../../../../../diagnostics_channel')
+const iastSetCookieChannel = dc.channel('datadog:iast:set-cookie')
```

Comment:
The channel could be defined in [packages/dd-trace/src/appsec/channels.js](https://github.com/DataDog/dd-trace-js/blob/61a15125a5685926413739521598ae23b59ff3bc/packages/dd-trace/src/appsec/channels.js#L6)